server/services/interventionsService.ts

```diff
@@ -599,4 +605,18 @@ export default class InterventionsService {
       query: { ...paginationParams },
     })) as Page<CaseNote>
   }
+
+  async addCaseNotes(token: string, createCaseNoteParam: CreateCaseNoteParams): Promise<CaseNote> {
+    const restClient = this.createRestClient(token)
+    const createCaseNoteDTO: Record<string, unknown> = {
```

Comment:
Why do we need the type assertion here (`Record<string, unknown>`)? Also, is it not a `CreateCaseNoteParams`?

Reply:
Ah I think this was removed in a subsequent commit, don't worry.